phel.html.markdown

```diff
@@ -0,0 +1,337 @@
+---
+language: phel
+filename: learnphel.phel
+contributors:
+    - ["Chemaclass", "https://github.com/Chemaclass"]
+---
+
+[Phel](https://phel-lang.org/) is a functional programming language that compiles to PHP.
+It is a dialect of Lisp inspired by Clojure and Janet.
+
+## Features
+- Built on PHP's ecosystem
+- Good error reporting
+- Persistent Datastructures (Lists, Vectors, Maps and Sets)
+- Macros
+- Recursive functions
+- Powerful but simple Syntax
+- REPL
+
+```phel
```

Comment:
I think pygments will probably choke on this; know any other lisps that use `#` for comments that you could sub in?

Reply:
CC: @jenshaase. Do you know something that could help us with that question?

> Or maybe we should really consider allowing `;` as comments as well in phel, so we can easily use the `clojure`/`lisp` syntax colouring everywhere 😂

Reply:
There are a few lisp that use `#` for comments. However, in `pygments` there is currently only `newlisp` that supports this. I took a look at the demo (https://pygments.org/demo/) and the highlighting looks ok to me. There is only a small problem with backslashes

Reply:
Right, everything looks good using `newlisp` except the backslashes... but I think it would be fine for now, don't you think? Do you want me to change `phel` to `newlisp` here (line 20)? I am fine with that, unless you have something different in your mind.
<img width="931" alt="Screenshot 2022-01-19 at 09 40 13" src="https://user-images.githubusercontent.com/5256287/150094404-f03a8177-d073-4b44-acc6-834177f28c70.png">